app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,164 @@
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.05" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_end"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.95" />
+
     <TextView
+        android:id="@+id/text_settings"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        style="@style/TextAppearance.AppCompat.White24"
+        android:text="@string/settings"
+        app:layout_constraintBottom_toBottomOf="@id/background_my_profile"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toTopOf="@id/background_my_profile"
+        app:layout_constraintVertical_bias="0.08" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/button_log_out"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        style="@style/ButtonStyle.LogOut"
+        app:cornerRadius="6dp"
+        android:text="@string/log_out"
+        app:layout_constraintBottom_toBottomOf="@id/background_my_profile"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintTop_toTopOf="@id/background_my_profile"
+        app:layout_constraintVertical_bias="0.067"
+        app:strokeColor="@color/log_out"
+        app:strokeWidth="2dp" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/view_my_contacts"
+        android:layout_width="0dp"
+        android:layout_height="55dp"
+        style="@style/ButtonStyle.Contacts"
+        android:text="@string/view_my_contacts"
+        app:cornerRadius="6dp"
+        android:textStyle="bold"
         app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/background_my_profile"
+        app:layout_constraintVertical_bias="0.95" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/edit_profile"
+        android:layout_width="0dp"
+        android:layout_height="40dp"
+        android:minHeight="48dp"
+        style="@style/ButtonStyle.EditProfile"
+        android:text="@string/edit_profile"
+        app:cornerRadius="6dp"
+        app:layout_constraintBottom_toTopOf="@id/view_my_contacts"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/background_my_profile"
+        app:layout_constraintVertical_bias="0.95"
+        app:strokeColor="@color/black"
+        app:strokeWidth="2dp" />
+
+    <LinearLayout
```

Comment:
Try to replace this linearlayout with constraintlayout flow and place elements in flow using flow_horizontalStyle attribute.

Reply:
done